qa/workunits/cephadm/nvme_initiator.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash -ex
```

Comment:
If you want `-ex` behavior, it's better to use `set -ex` in the body so that it takes effect independent of how the script is executed (`./foo.sh` vs `bash ./foo.sh`, etc).